sdk/search/search-documents/src/indexModels.ts

```diff
@@ -604,7 +604,7 @@ export type IndexDocumentsAction<T> = {
    * The operation to perform on a document in an indexing batch. Possible values include:
    * 'upload', 'merge', 'mergeOrUpload', 'delete'
    */
-  __actionType: IndexActionType;
+  actionType: IndexActionType;
```

Comment:
yeah this change feels risky as `actionType` doesn't seem implausible to exist on user documents

Reply:
same as above